tree/10_ncf_internals/modules/templates/jinja2-templating.py

```diff
@@ -67,6 +74,24 @@ def render(opts, args):
     if opts.strict:
         env.undefined = StrictUndefined
 
+    # Register customs
+    sys.path.append('/var/rudder/ncf/local/10_ncf_internals/modules/templates/')
+
+    if pkgutil.find_loader('jinja2_custom') is None:
+        custom_filters = False
+    else:
+        custom_filters = True
```

Comment:
this is one line: 
custom_filters = pkgutil.find_loader('jinja2_custom') is not None